app/models/submission.rb

```diff
@@ -5,6 +5,7 @@ class Submission < ApplicationRecord
 
   belongs_to :local_authority
   has_one :planning_application, dependent: :nullify
+  has_one :case_record, dependent: :nullify
```

Comment:
This makes me think we should aim to migrate PlanningApplication to CaseRecord soon, but that's going to be fiddly. Started testing it out in #2416.